mmvstr.c

```diff
@@ -138,25 +137,23 @@ void let(vstring *target, vstring source) {
     if (targetLength) {
       free(*target);
     }
-    *target= "";
+    *target = "";
   }
 
   freeTempAlloc(); /* Free up temporary strings used in expression computation */
 
 } /* let */
 
-vstring cat(vstring string1,...)        /* String concatenation */
```

Comment:
cat has another limitation not mentioned in the comments of master: the list of sources must not be empty i.e cat(NULL) is illegal